decompressed/base/usr/sbin/mount_modoverlay

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+[ -f "/rom/$(command -v $0)" ] && . /rom/$(command -v $0)
```

Comment:
L'hook per le release 16.2 potrebbe essere un eseguibile binario e non uno script, ergo il source con "." non lo si potrebbe fare. Provate direttamente ad eseguirlo senza "."
Subito sotto conviene salvare local ret=$# in una variabile e mettere in fondo a questo file un return $ret

Reply:
e gli vanno passati anche i parametri $@

Reply:
nsomma, sostituisco la riga con `ret=$("$0" $@)` e in fondo `return $ret`

Reply:
no, non mi serve l'output, server solo l'exit code

Reply:
```bash
[ -f "/rom/$(command -v $0)" ] && /rom/$(command -v $0) $@
local ret=$?
…
return $ret
```